packages/crayons-core/src/components.d.ts

```diff
@@ -1967,6 +1967,10 @@ export namespace Components {
           * Whether clicking on option selects it.
          */
         "allowSelect": boolean;
+        /**
+          * Third line text in conversation can be author details etc.
+         */
+        "author": Author;
```

Comment:
@harshith-venkatesh-freshworks Can the prop name be more generic like metaText or tertiaryText as author is very specific and it can be anything in a conversation item. wdyt?

Reply:
@rihansiddhi , sure will go ahead and name it as metaText/tertiaryDetails itself